### PR TITLE
Add new start_listening2 API for Routing.

### PR DESCRIPTION
**Pull request by ned14**
[Review on Reviewable](https://reviewable.io/reviews/maidsafe/crust/123)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @chandraprakash

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "crust"
-version = "0.0.60"
+version = "0.0.61"
```
Comment (chandraprakash) [inline Cargo.toml]: Can you drop the version bump please. Need to do in a separate pull request.
